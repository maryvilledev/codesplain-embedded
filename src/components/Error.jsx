import React, { PropTypes } from 'react';

const style = {
  border: ' 8px solid #FF8080',
  textAlign: 'center',
  padding: '10px',
  paddingBottom: '20px',
  borderRadius: '10px',
}

const Error = ({ snippetKey }) => (
  <div style={style}><h1>Codesplain Error</h1>Could not load {snippetKey}.</div>
)

export default Error;
