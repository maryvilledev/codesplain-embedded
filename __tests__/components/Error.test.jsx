import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json'

import Error from '../../src/components/Error';

describe('<Error />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Error snippetKey="birdperson.py"/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
