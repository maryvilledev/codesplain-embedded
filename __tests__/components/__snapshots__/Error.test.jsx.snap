// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> matches snapshot 1`] = `
<div
  style={
    Object {
      "border": " 8px solid #FF8080",
      "borderRadius": "10px",
      "padding": "10px",
      "paddingBottom": "20px",
      "textAlign": "center",
    }
  }
>
  <h1>
    Codesplain Error
  </h1>
  Could not load 
  birdperson.py
  .
</div>
`;
